config/database.yml

```diff
@@ -1,8 +1,14 @@
+# If you want to change this file, please keep the changes in your working
+# copy by using
+#
+#     git update-index --skip-worktree config/database.yml
```

Comment:
BTW, maybe we should just create a wiki or doc in markdown somewhere like `How to override default DB configuration` instead of putting all this stuff in the code